_dashboards/query-workbench.md

```diff
@@ -0,0 +1,233 @@
+
+2. Run a query.
+    - Select the **SQL** button. In the query editor, type a SQL expression and then select the **Run** button to run the query. 
+    
+    The following is an example query that retrieves the first name, last name, and balance from the `accounts` index for accounts with a balance greater than 10,000 and sorts by balance in descending order:
+
+    ```sql
+    SELECT
+      firstname,
+      lastname,
+      balance
+    FROM
+      accounts
+    WHERE
+      balance > 10000
+    ORDER BY
+      balance DESC;
+    ```
+    {% include copy.html %}
+    
+3. Analyze the results.
+    - View the results in the **Results** pane, which presents the query output in tabular format. You can sort, filter, and download the results as needed.
+
+   The following image shows the query editor pane and results pane for the preceding SQL query:
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-workbench-query-step2.png" alt="Query Workbench SQL query input and results output panes" width="800">
+
+4. Clear the query editor.  
+    - Select the **Clear** button to clear the query editor and run a new query. 
+
+5. Examine how the query is processed.
+    - Select the **Explain** button to examine how OpenSearch processes the query, including the steps involved and order of operations. 
+
+    The following image shows the explanation of the SQL query that was run in step 2.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-explain.png" alt="Query Workbench SQL query explanation pane" width="500">
+
+### Viewing indexes
+
+To view a list of all your indexes, run the following `SHOW` statement:
+
+```sql
+SHOW TABLES LIKE %
+```
+{% include copy.html %}
+
+The following image shows an example of indexes listed in the **Results** pane:
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-list.png" alt="Query Workbench list of indexes view within the Results pane" width="800">
+
+### Retrieving data
+
+To retrieve a document, run a `SELECT` statement with a `FROM` clause in the Query Workbench editor. The following is the basic SQL syntax to retrieve a document in OpenSearch: 
+
+```sql
+SELECT *
+FROM <index_name>
+WHERE <field_name> = <field_value>
+```
+{% include copy.html %}
+
+In this syntax, replace `<index_name` with the name of the index where the document is stored, `<field_name` with the name of the field you want to search, and `<field_value` with the value you want to search for. The `*` in the `SELECT` statement means that you want to retrieve all fields from the document. If you only want to retrieve specific fields, replace `*` with a comma-separated list of field names. 
+
+### Deleting data
+
+To delete a document from an index, run a `DELETE` statement with the `WHERE` clause. The following is the basic SQL syntax to delete a document from an index: 
```

Comment:
Same as above. The WHERE clause can be more complicated. Maybe remove this section altogether and make the previous section parallel to PPL?

Reply:
Deleted the SQL sections about listing, retrieving, and deleting. If a developer wants to include this moving forward, we can have them write that information from the dev POV.